cli/templates/checklist/checklist.html

```diff
@@ -37,13 +37,12 @@ <h3>Before the Release (Tuesday)</h3>
 <h3>Create the Release{{ if .Scheduled }} (Thursday) {{end}}</h3>
 <!-- /wp:heading -->
 
-{{ Task `Verify that <code>gutenberg-mobile/RNTAztecView.podspec</code> and <code>gutenberg-mobile/gutenberg/packages/react-native-aztec/RNTAztecView.podspec</code>
- refer to the same <code>WordPress-Aztec-iOS</code> version and are pointing to a stable, tagged release (e.g. 1.14.1). If they are not, we may need to <a href="#create-a-new-aztec-release">create a new Aztec</a> release.` }}
+{{ Task `Clone the <a href="https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile">CLI</a> or pull the latest version if you have already cloned it.` }}
```

Comment:
We shouldn't use clones of the CLI. Instead we should suggest installing the [latest release](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/releases)